netman/api/switch_api.py

```diff
@@ -37,6 +37,8 @@ def hook_to(self, server):
         server.add_url_rule('/switches/<hostname>/vlans/<vlan_number>/ips/<path:ip_network>', view_func=self.remove_ip, methods=['DELETE'])
         server.add_url_rule('/switches/<hostname>/vlans/<vlan_number>/vrrp-groups', view_func=self.add_vrrp_group, methods=['POST'])
         server.add_url_rule('/switches/<hostname>/vlans/<vlan_number>/vrrp-groups/<vrrp_group_id>', view_func=self.remove_vrrp_group, methods=['DELETE'])
+        server.add_url_rule('/switches/<hostname>/vlans/<vlan_number>/varp/<path:ip_network>', view_func=self.add_varp_ip, methods=['POST'])
```

Comment:
this is not coherent with the internap structure.

The internap structure is a list called "varp-ips"
and the rest says there's a varp object of which its ID is a the IP.